build/media-suite_compile.sh

```diff
@@ -1709,6 +1709,13 @@ if [[ $ffmpeg != "no" ]]; then
             do_patch "https://0x0.st/zeB6.txt"
         fi
 
+        if enabled libsvthevc && [[ $bits != "32bits"]]; then
```

Comment:
Then you can leave just the check for `enabled libsvthevc` here.

Reply:
Done